src/ar/tuktukcinema/src/eu/kanade/tachiyomi/animeextension/ar/tuktukcinema/Tuktukcinema.kt

```diff
@@ -151,6 +162,13 @@ class Tuktukcinema : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     private fun extractVideos(url: String): List<Video> {
         return when {
+            url.contains("egtpgrvh") -> {
+                val videoList = mutableListOf<Video>()
+                val request = client.newCall(GET(url, headers)).execute().asJsoup()
+                val data = request.selectFirst("script:containsData(m3u8)")!!.data()
+                val masterUrl = data.substringAfter("sources: [{").substringAfter("file:\"").substringBefore("\"}")
+                PlaylistUtils(client, headers).extractFromHls(masterUrl)
```

Comment:
Might as well use the instance that were lazily created earlier
```suggestion
                playlistUtils.extractFromHls(masterUrl)
```